static/app/views/profiling/content.tsx

```diff
@@ -28,7 +33,22 @@ interface ProfilingContentProps {
 function ProfilingContent({location, selection}: ProfilingContentProps) {
   const organization = useOrganization();
   const cursor = decodeScalar(location.query.cursor);
-  const [requestState, traces, pageLinks] = useProfiles({cursor, selection});
+  const profileFilters = useProfileFilters();
+  const query = decodeScalar(location.query.query);
+  const [requestState, traces, pageLinks] = useProfiles({cursor, query, selection});
+
+  const handleSearch = (searchQuery: string) => {
```

Comment:
Thoughts on useCallback here :D? It looks like it could be a stable reference for the entire lifetime. We should also just tie it to the SmartSearchBarProps by typing it as

```suggestion
  const handleSearch: SmartSearchBarProps["onSearch"]  = (searchQuery) => {
```